### ISSUE TITLE
Cypress run crashes on finish

### ISSUE THREAD

**Issue by aldarund** [cypress-io/cypress#2044]
### Current behavior:

```
    Spec                                                Tests  Passing  Failing  Pending  Skipped
  ┌────────────────────────────────────────────────────────────────────────────────────────────────┐
  │ √ index.js                                  00:02        1        1        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ √ signup.js                                 00:02        2        2        -        -        - │
  └────────────────────────────────────────────────────────────────────────────────────────────────┘
    All specs passed!                           00:05        3        3        -        -        -

events.js:167
      throw er; // Unhandled 'error' event
      ^

Error: shutdown EPIPE
    at ReadStream.Socket._final (net.js:366:25)
    at callFinal (_stream_writable.js:612:10)
    at process._tickCallback (internal/process/next_tick.js:63:19)
Emitted 'error' event at:
    at ReadStream.onerror (_stream_readable.js:687:12)
    at ReadStream.emit (events.js:182:13)
    at emitErrorNT (internal/streams/destroy.js:82:8)
    at emitErrorAndCloseNT (internal/streams/destroy.js:50:3)
    at process._tickCallback (internal/process/next_tick.js:63:19)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

### Desired behavior:
No crash :)

### Versions
Cypress 3.0.1 Windows 10 Node 10.5.0

**Comment by jennifer-shehane** [comment]
Duplicate of https://github.com/cypress-io/cypress/issues/1841 

The code for this is done, but this has yet to be released. We'll update this issue and reference the [changelog](https://on.cypress.io/changelog) when it's released.

**Comment by brian-mann** [comment]
Released in [`3.0.2`](https://docs.cypress.io/guides/references/changelog.html#3-0-2).
